CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/GiniConfiguration.swift

```diff
@@ -495,8 +495,7 @@ import GiniBankAPILibrary
 
     var documentService: DocumentServiceProtocol?
 
-    // swiftlint:disable function_body_length
-    // swiftlint:disable function_parameter_count
+    // swiftlint: disable function_parameter_count
```

Comment:
To keep the consistent style of the `swiftlint` messages - no space or shall we add a space after the colon
```suggestion
    // swiftlint:disable function_parameter_count
```